docs/installation.rst

```diff
@@ -368,11 +368,19 @@ A `conda-lock`_ file is a cross platform way of specifying a conda environment t
 Unlike the single file installer, an internet connection is required to install from a ``conda-lock`` file.
 We recomend the use of a ``conda-lock`` file when the same conda environment is required across different systems.
 
-See https://github.com/conda/conda-lock?tab=readme-ov-file#installation for instructions on how to install ``conda-lock``.
 
 .. note::
 
-   You will likely need to install ``conda-lock``
```

Comment:
Could you change the order of this file to put conda-lock before the single file installer please?

Reply:
Yes! I did that on the indy benchmark PR but wasn't sure we wanted to also do that on our main docs page, I have no objections to moving it!

Reply:
Again, was looking at the wrong thing!